org.yakindu.sct.examples.fmu.bouncingball/.cproject

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The builder builds packages a dll/dylib with name `org.yakindu.sct.examples.fmu.bouncingball.dll` while the fmu is named `bouncingball.fmu.` Does this comply to the FMI standard? (Not tested by review)

Reply:
in my case the FMU is named "org.yakindu.sct.examples.fmu.bouncingball.fmu"